apps/frontend/src/config.ts

```diff
@@ -0,0 +1,15 @@
+let baseUrl: string;
+switch (window.location.host.split('.').length) {
```

Comment:
This is ... less than ideal, ideas @fampinheiro?
I also don't really wanna pass the base URL via the environment when building though, that'd mean the same commit builds a different frontend bundle on `master` vs a branch, which is also weird

Reply:
Apart from an environment variable I'm not seeing many options.